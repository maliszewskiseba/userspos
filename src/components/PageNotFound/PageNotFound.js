import React from 'react';
import i18n from '../../i18n';

const PageNotFounded = () => <h2>{i18n.t('custom.notFound')}</h2>;

export default PageNotFounded;
