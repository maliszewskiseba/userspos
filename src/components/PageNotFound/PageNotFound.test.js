import React from "react";
import { create } from "react-test-renderer";
import PageNotFound from "./PageNotFound";

describe("Loading PageNotFound component", () => {
  test("it matches the snapshot", () => {
    const component = create(<PageNotFound />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
