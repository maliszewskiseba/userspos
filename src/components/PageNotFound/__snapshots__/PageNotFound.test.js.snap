// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading PageNotFound component it matches the snapshot 1`] = `
<h2>
  Page not found
</h2>
`;
